src/main/java/qna/domain/Answer.java

```diff
@@ -3,15 +3,29 @@
 import qna.NotFoundException;
 import qna.UnAuthorizedException;
 
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Lob;
+import javax.persistence.Table;
 import java.util.Objects;
 
-public class Answer {
-    private Long id;
-    private Long writerId;
-    private Long questionId;
+@Entity
+@Table(name = "answer")
+public class Answer extends CommonEntity {
+    @Lob
     private String contents;
+
+    @Column(nullable = false)
     private boolean deleted = false;
 
+    private Long writerId;
+
+    private Long questionId;
+
+    public Answer() {
+
+    }
```

Comment:
엔티티의 접근제어자를 protected로 변경해보는건 어떨까요 ?
아래 링크가 도움이 되었으면 해요 :)
https://wbluke.tistory.com/6
```suggestion
    protected Answer() {

    }
```

Reply:
앗, 이건. 놓친 부분입니다. ㅠ 다음 리뷰요청시 수정하도록 할께요.